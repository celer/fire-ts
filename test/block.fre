<!--%{Header}
	This is a block
}%-->

//%{header}
	Wolf
//}%

Hello <%=pid%>
