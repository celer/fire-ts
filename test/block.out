<!--%{Header}
	This is a block
}%-->

//%{header}
FOOFOO
//}%

Hello 33
